Include/new_geometry.h

```diff
@@ -0,0 +1,40 @@
+/**
+ * @file new_geometry.h
+ * @brief Functions needed for the new geometry implementation that
+ *        will replace the current geometry in the future
+ */
+
+ #ifndef NEW_GEOMETRY_H
+ #define NEW_GEOMETRY_H
+
+static inline int lexi(int b0, int b1, int b2, int b3, int x0, int x1, int x2, int x3);
```

Comment:
static functions must not be included in the header file.
From this file we only need the 5 functions at the bottom after the stddef include.